modules/mono/glue/GodotSharp/GodotSharp/Core/DelegateUtils.cs

```diff
@@ -76,6 +76,9 @@ private enum TargetKind : uint
 
         internal static bool TrySerializeDelegate(Delegate @delegate, Collections.Array serializedData)
         {
+			if(@delegate is null)
```

Comment:
Style issue (please see surrounding code and use the same style).

Reply:
Line 94 and 97 share the same style without braces, but maybe this is better :)

Reply:
I think Akien meant:

```suggestion
			if (@delegate is null)
```